.editorconfig

```diff
@@ -0,0 +1,60 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# If you are using VS Code consider using the following extension
+# https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
+
+# The option max_line_length is only supported by some editors. Please read the official documentation.
+# https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length
+
+# This file is based on the .editorconfig of the apluslms/a-plus project. Therefore, all changes implemented in this
+# particular file should be considered to be applied also in the base .editorconfig file.
+# https://github.com/apluslms/a-plus/blob/master/.editorconfig
```

Comment:
The new upstream is https://github.com/apluslms/apluslms.github.io/blob/master/.editorconfig (@raphendyr just did it now)

There are some differences between these files now. Could you unify this one with the upstream? If there are local changes for this course-templates project, they should be added to the end of the file.